packages/components/src/components/Dropdown/styles/dropdown.module.scss

```diff
@@ -79,15 +78,11 @@
         color: var(--text-color-disabled);
     }
 
-    .itemContent {
+    span {
```

Comment:
What if the children is a `p` or `div` tag?

Reply:
that would be a custom element, we shouldn't interfere there. This is affecting the text passed in like suggested in our examples / the migration guide